datafusion/sqllogictest/test_files/string/dictionary_utf8.slt

```diff
@@ -64,6 +64,22 @@ _ (empty) false false false false
 NULL NULL NULL NULL NULL NULL
 NULL NULL NULL NULL NULL NULL
 
+# TODO: move it back to `string_query.slt.part` after fixing the issue
+# see issue https://github.com/apache/datafusion/pull/13221
```

Comment:
https://github.com/apache/datafusion/pull/13221 is now fixed, so I think we can move this test to `string_query.slt`

that being said, this seems like it is an improvement to me

Reply:
oops, it's my mistake. It should wait for https://github.com/apache/datafusion/issues/13329 actually. I pasted the wrong issue here 😢 . Thanks for reminding it.